### ISSUE TITLE
Relation nodes should be blue

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/Brainweaver#112]
* Start BrainweaverDeveloper
 * Click `QtConceptMap`

then the node on the edge is white instead of blue:

![issue](https://cloud.githubusercontent.com/assets/2098230/20029844/727920d4-a357-11e6-97e8-5794c6f54404.png)

It should be blue

**Comment by richelbilderbeek** [comment]
```
void ribi::cmap::QtConceptMap::SetMode(const ribi::cmap::Mode mode) noexcept
{
  m_mode = mode;
  auto qtnodes = GetQtNodes(GetScene()); //Should use GetQtNodesAlsoOnQtEdge? 
 for (auto qtnode: qtnodes)
  {
    const auto f = GetQtNodeBrushFunction(m_mode);
    qtnode->SetBrushFunction(f);
  }
}
```

**Comment by richelbilderbeek** [comment]
![fixed](https://cloud.githubusercontent.com/assets/2098230/20209870/e6ad07b0-a7f6-11e6-8036-5aad92ed1884.png)

**Comment by richelbilderbeek** [comment]
Fixed with 0277883b21bcaa2f9c7991b5823547e4c314fb16 , but sadly without a test to secure this. define `CAN_TEST_112` to try to get the test working.
